apps/dashboard/tests/code-created-flow.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { test } from '@playwright/test';
```

Comment:
How about naming it from the user journey perspective? For example: `sync-workflow.e2e.ts.`